src/Integration.Vsix/SonarLintTagger/IssueToIssueMarkerConverter.cs

```diff
@@ -51,30 +50,15 @@ public IssueMarker Convert(IAnalysisIssue issue, ITextSnapshot textSnapshot)
             {
                 throw new ArgumentNullException(nameof(issue));
             }
+
             if (textSnapshot == null)
             {
                 throw new ArgumentNullException(nameof(textSnapshot));
             }
 
-            // SonarLint issues line numbers are 1-based, spans lines are 0-based
-
             var span = issueSpanCalculator.CalculateSpan(issue, textSnapshot);
 
-            // A start line of zero means the issue is file-level i.e. not associated with a particular line
-            if (issue.StartLine == 0)
```

Comment:
Logic moved to `IssueSpanCalculator`, so it could also be re-used in [document navigation PR](https://github.com/SonarSource/sonarlint-visualstudio/blob/ccc322205d6caa045705b33432b726397260915c/src/IssueViz/Selection/LocationChangedEventListener.cs#L66)